### PR TITLE
Update example mysql confs

### PR DESCRIPTION
**Pull request by aweaver89**
Update config examples for the mysql integration to match the current values accepted by the integration

#### What this PR does / why we need it:

Updates the examples of the mysql integration to correspond to the current values. For example replace the server parameter with the current host that is in the integration example. Also removed the "" around the server and password. This will hopefully help avoid customers using these in their setups and having issues.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Documentation has been updated with helm-docs (run: .github/helm-docs.sh)
- [x] Chart Version bumped
- [x] CHANGELOG.md has been updated
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by clamoriniere — COMMENTED** [review]
Hi @aweaver89 
thanks for opening this Pull request.

Could you please follow the pull request check list so we can validate and merge it. 🙇

**Comment by aweaver89** [comment]
@clamoriniere - I've removed some of the other checks because they don't seem to be relevant since this is more of a documentation update. Do I need to update the changelog for these minor changes? The version bump seems to be mandatory for the CI tests, but this is really minor imho.

**Comment by clamoriniere** [comment]
The CI tooling is not able to differentiate fix,feature against documentation Pull Request. So unfortunately even for documentation we need to follow the same PR checklist.

**Comment by aweaver89** [comment]
@clamoriniere - Everything passed this time 🥳

File: charts/datadog/templates/cluster-agent-scc.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ allowHostIPC: false
 allowHostPID: false
 allowPrivilegeEscalation: false
 allowPrivilegedContainer: false
+allowedCapabilities: null
+defaultAddCapabilities: null
```
Comment (clamoriniere) [inline charts/datadog/templates/cluster-agent-scc.yaml:21]: This 2 lines should not be added back

**Review by clamoriniere — CHANGES_REQUESTED** [review]
I think you have a merged when you have rebased your pr

File: charts/datadog/templates/cluster-agent-scc.yaml (hunk 1 above)
Comment (aweaver89) [reply charts/datadog/templates/cluster-agent-scc.yaml:21]: ahhhh I thought that you had added those. It was a bit weird with your merge earlier today ;)

Comment (aweaver89) [reply charts/datadog/templates/cluster-agent-scc.yaml]: @clamoriniere - fixed this now 👍

**Review by clamoriniere — APPROVED** [review]

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**